WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -37,6 +37,7 @@ final class OrderListViewController: UIViewController, GhostableViewController {
 
     /// The data source that is bound to `tableView`.
     private lazy var dataSource: UITableViewDiffableDataSource<String, FetchResultSnapshotObjectID> = {
+        self.loadViewIfNeeded()
```

Comment:
Added this to make sure IBOutlets are loaded, since tests were failing because `self.tableView` was nil at the moment of creating the `dataSource`

Reply:
Should we add a comment in the code explaining why are we doing this? I'm not sure what could be the alternative (if any), but, are there any concerns regarding causing unexpected side-effects with the view lifecycle by calling `loadViewIfNeeded` in the data source initialization here?

Reply:
The only side effect I see that the view hierarchy is loaded/created sooner, but in our normal flow of the app calling this will not do anything because view will already be loaded, it is just needed for use cases (like tests) when the view is not yet used so the table view is not connected to the IBOutlet etc.

Reply:
But I will add comment (similar to commit message) just so we know in the future